Documentation/sources/troubleshooting/validate-messages-in-griffon.md

```diff
@@ -1,4 +1,4 @@
-# Validate in-app messaging using AEPAssurance SDK extension and the Assurance UI
+# Validate In-App Messaging using Assurance
```

Comment:
in-app messaging shouldn't be capitalized, it's not a proper noun :)

Reply:
reverting 👍🏼